tests/io/test_cif.py

```diff
@@ -735,10 +735,10 @@ def test_bad_cif(self):
         filepath = f"{TEST_FILES_DIR}/cif/bad_occu.cif"
         parser = CifParser(filepath)
         with pytest.raises(
-            ValueError, match="No structure parsed for section 1 in CIF.\nSpecies occupancies sum to more than 1!"
```

Comment:
Previous error message might be confusing, as this fails because the occupancy is greater than `1 + tolerance`, not just `1`.